package.json

```diff
@@ -8,22 +8,38 @@
   "license": "Apache-2.0",
   "scripts": {
     "compile": "tsc",
-    "lint": "tslint --project .",
     "test": "ava",
     "clean": "gts clean",
-    "fix": "gts fix"
+    "cli": "node ./build/src/cli/main.js",
+    "lint": "eslint './src/**/*.ts' './test/**/*.ts'"
```

Comment:
Could you lint '**/*.ts' to catch all Typescript files?

Reply:
Catching all `.ts` files seems to catch the `.d.ts` files in `build/`.  I could do `eslint './test/**/*.ts' --ignore-pattern '**/*.d.ts'`, but I'm not sure that's much better than what I have now.  (This comment probably belongs in #39 but the diff is messy - my bad for making dependent PRs.  See description)